ListForForceCompletion.py

```diff
@@ -0,0 +1,411 @@
+,'task_EXO-RunIISummer15wmLHEGS-06119'
+,'task_EXO-RunIISummer15GS-11292'
+,'task_HIG-RunIIFall17wmLHEGS-01594'
+,'task_HIG-RunIIFall17wmLHEGS-01610'
+,'task_HIG-RunIIFall17wmLHEGS-01495'
+,'task_HIG-RunIIFall17wmLHEGS-01507'
+,'task_HIG-RunIIFall17wmLHEGS-01523'
+,'task_EXO-RunIISummer15wmLHEGS-06232'
+,'task_EXO-RunIISummer15GS-11105'
+,'task_B2G-RunIISummer15wmLHEGS-01674'
+,'task_EXO-RunIISummer15wmLHEGS-06281'
+,'task_BTV-RunIIFall17DRPremix-00096'
+,'task_EXO-RunIISummer15GS-11376'
+,'task_HIG-RunIIFall17wmLHEGS-01612'
+,'task_HIG-RunIIFall17wmLHEGS-01919'
+,'task_EXO-RunIISummer15wmLHEGS-06133'
+,'task_EXO-RunIISummer15wmLHEGS-06148'
+,'task_EXO-RunIISummer15wmLHEGS-06223'
+,'task_EXO-RunIIFall17wmLHEGS-00143'
+,'task_EXO-RunIIFall17GS-00939'
+,'task_HIG-RunIIFall17wmLHEGS-02311'
+,'task_EXO-RunIISummer15GS-11372'
+,'task_EXO-RunIISummer15GS-11327'
+,'task_EXO-RunIISummer15GS-11337'
+,'task_EXO-RunIISummer15GS-11256'
+,'task_B2G-RunIISummer15wmLHEGS-01666'
+,'task_EXO-RunIISummer15wmLHEGS-06117'
+,'task_EGM-RunIISummer18GS-00023'
+,'task_EXO-RunIISummer15wmLHEGS-06312'
+,'task_EXO-RunIISummer15wmLHEGS-05964'
+,'task_EXO-RunIISummer15wmLHEGS-06248'
+,'task_EXO-RunIIFall17GS-01101'
+,'task_SUS-RunIIFall17wmLHEGS-00183'
+,'task_HIG-RunIIFall17wmLHEGS-01502'
+,'task_B2G-RunIISummer15wmLHEGS-01662'
+,'task_HIG-RunIIFall17wmLHEGS-01362'
+,'task_EXO-RunIISummer15GS-11309'
+,'task_HIG-RunIIFall17wmLHEGS-00873'
+,'task_SUS-RunIIFall17wmLHEGS-00057'
+,'task_HIG-RunIIFall17wmLHEGS-00860'
+,'task_EXO-RunIIFall17GS-00546'
+,'task_EXO-RunIISummer15wmLHEGS-06086'
+,'task_EXO-RunIISummer15wmLHEGS-06143'
+,'task_EXO-RunIISummer15wmLHEGS-06229'
+,'task_EXO-RunIISummer15GS-11357'
+,'task_EXO-RunIISummer15GS-11278'
+,'task_EXO-RunIISummer15GS-11294'
+,'task_SUS-RunIIFall17wmLHEGS-00017'
+,'task_HIG-RunIIFall17wmLHEGS-01583'
+,'task_EXO-RunIISummer15wmLHEGS-06187'
+,'task_TOP-RunIIFall17wmLHEGS-00022'
+,'task_EXO-RunIISummer15wmLHEGS-06018'
+,'task_EXO-RunIISummer15wmLHEGS-06251'
+,'task_HIG-RunIIFall17wmLHEGS-01649'
+,'task_HIG-RunIIFall17wmLHEGS-01863'
+,'task_SUS-RunIIFall17wmLHEGS-00165'
+,'task_EXO-RunIIFall17wmLHEGS-00303'
+,'task_EXO-RunIISummer15wmLHEGS-06057'
+,'task_EXO-RunIISummer15wmLHEGS-06193'
+,'task_EXO-RunIISummer15wmLHEGS-06253'
+,'task_EXO-RunIISummer15wmLHEGS-06221'
+,'task_EXO-RunIISummer15wmLHEGS-06041'
+,'task_EXO-RunIIFall17wmLHEGS-00284'
+,'task_EXO-RunIISummer15wmLHEGS-06049'
+,'task_EXO-RunIISummer15GS-11276'
+,'task_HIG-RunIIFall17wmLHEGS-01715'
+,'task_TOP-RunIIFall17wmLHEGS-00180'
+,'task_HIG-RunIIFall17GS-00111'
+,'task_EXO-RunIISummer15wmLHEGS-06256'
+,'task_EXO-RunIISummer15wmLHEGS-06075'
+,'task_HIG-RunIIFall17wmLHEGS-01609'
+,'task_EXO-RunIISummer15wmLHEGS-06052'
+,'task_EXO-RunIIFall17wmLHEGS-00201'
+,'task_EXO-RunIIFall17GS-00231'
+,'task_EXO-RunIISummer15GS-11264'
+,'task_B2G-RunIISummer15wmLHEGS-01650']
+
+
+print(' Prepid       Dataset name       Completed events   Total events      Link to MCM')
+
+for i in List:
```

Comment:
What is the purpose of this script? If "List" is there only as an example, maybe it can be shorter?